openquakeplatform_taxtweb/templates/taxtweb/index.html

```diff
@@ -24,6 +24,9 @@
     {{block.super}}
 
 {% include "taxtweb/incl_taxtweb_inlinecss.html" %}
+ <div id="oq-body-heading">
+        <!--<h1>TaxTweb - Graphical tool for editing GEM Taxonomy strings</h1> -->
```

Comment:
Commented title ?

Reply:
This must be deleted

Reply:
Solved with this https://github.com/gem/oq-platform-taxtweb/pull/27/commits/f1a50f79420797526eed5f9c4e4bf8b88de0b3bf